navigator_engine/tests/unit_tests/test_data_loaders.py

```diff
@@ -186,3 +186,55 @@ def test_load_estimates_dataset_without_workflow_state(mock_engine, mocker):
             'data': {'completedTasks': []}
         }
     }
+
+
+def test_load_csv_from_zipped_resource(mock_engine, mocker):
+
+    with open('navigator_engine/tests/test_data/test_spectrum_file.pjnz', 'rb') as f:
+        spectrum_file = f.read()
+
+    mock_load_estimates_dataset_resource = mocker.patch(
+        'navigator_engine.pluggable_logic.data_loaders.load_estimates_dataset_resource',
+        return_value={
+            'spectrum-file': {
+                'data': spectrum_file,
+                'auth_header': 'Bearer xxxx-xxxx-xxxx-xxxx',
```

Comment:
In CKAN's case it is not a bearer token.  I would just get rid of Bearer, although admittedly it's only a cosmetic change for readbility.